src/main/webapp/assets/js/analytics.js

```diff
@@ -2,8 +2,8 @@
   /* jshint  strict: true*/
   /* global $, Blob, document, window*/
 
-
   $(document).ready(function() {
+      var reportPollInterv = 10000; //poll every 10 sec
```

Comment:
When I use variables to hold a value for time, I like to name them so that the unit of time is fairly obvious.  E.g. since this is milliseconds, maybe `reportPollIntervalMs`, `reportPollIntervalMsec`, or `reportPollIntervalMillis`.